torch/_export/serde/schema.py

```diff
@@ -331,8 +331,8 @@ class GraphSignature:
 
 @dataclass
 class RangeConstraint:
-    min_val: int
-    max_val: int
+    min_val: Optional[int]
+    max_val: Optional[int]
```

Comment:
totally nit: I might be wrong, but does adding `Optional[int] = None` let you upgrade only the minor schema version? (provides default value)